packages/create-sitecore-jss/src/initializers/nextjs/index.ts

```diff
@@ -56,11 +65,12 @@ export default class NextjsInitializer implements Initializer {
       addInitializers = addInitAnswer.addInitializers;
     }
 
+    saveConfiguration([...addInitializers, ...args.templates], pkgPath);
```

Comment:
We can't run it at the end, after all initializers finished work, but after each - yes
because we want to know which add-ons will be installed, during the process, we don't have order
https://github.com/Sitecore/jss/pull/980/files#diff-fbe9091647ef0dc7fccc4699163effaf94f8b5fc692362292afccf44d2d94498